artemis-server/src/main/java/org/apache/activemq/artemis/core/deployers/impl/FileConfigurationParser.java

```diff
@@ -1261,6 +1265,14 @@ protected Pair<String, AddressSettings> parseAddressSettings(final Node node) {
             long pageSizeLong = ByteUtil.convertTextBytes(getTrimmedTextContent(child));
             Validators.POSITIVE_INT.validate(PAGE_SIZE_BYTES_NODE_NAME, pageSizeLong);
             addressSettings.setPageSizeBytes((int) pageSizeLong);
+         }  else if (MAX_READ_PAGE_MESSAGES_NODE_NAME.equalsIgnoreCase(name)) {
+            long maxReadPageMessages = Long.parseLong(getTrimmedTextContent(child));
+            Validators.POSITIVE_INT.validate(PAGE_SIZE_BYTES_NODE_NAME, maxReadPageMessages);
+            addressSettings.setMaxReadPageMessages((int)maxReadPageMessages);
+         }  else if (MAX_READ_PAGE_BYTES_NODE_NAME.equalsIgnoreCase(name)) {
+            long maxReadPageBytes = ByteUtil.convertTextBytes(getTrimmedTextContent(child));
+            Validators.POSITIVE_INT.validate(PAGE_SIZE_BYTES_NODE_NAME, maxReadPageBytes);
```

Comment:
Both the validate calls pass the same wrong name.

Reply:
wow... you're good man! thanks!